tests/commands/test_pack.py

```diff
@@ -72,6 +73,14 @@ def test_simple_succesful_build(tmp_path, caplog, bundle_yaml, config):
     expected = "Created '{}'.".format(zipname)
     assert [expected] == [rec.message for rec in caplog.records]
 
+    # check the created manifest for these particular values that depend on given info
+    manifest = yaml.safe_load(zf.read('manifest.yaml'))
+    assert manifest['architectures'] == [platform.machine()]
```

Comment:
I missed this in the previous PR, we want this to be the equivalent architecture entries for deb and snaps.

Reply:
This was addressed in the previous branch.